### PR TITLE
Add Urdu fake news dataset.

### PR DESCRIPTION
**Pull request by chaitnayabasava**
Added Urdu fake news dataset. More information about the dataset can be found <a href="https://github.com/MaazAmjad/Datasets-for-Urdu-news">here</a>.

### CODE REVIEW

File: datasets/urdu_fake_news/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+annotations_creators:
+  - expert-generated
+language_creators:
+  - expert-generated
+languages:
+  - ur
+licenses: []
```
Comment (lhoestq) [inline datasets/urdu_fake_news/README.md:8]: is there no license or is the license unknown ?
If it's unknown please use 
```
licenses:
- unknown
```

File: datasets/urdu_fake_news/urdu_fake_news.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+            name = os.path.basename(filename)
+            key = name.rstrip(".txt")
+
+            _class = 1 if ("Real" in filename) else 0
+            category = "".join([i for i in key if not i.isdigit()])
+            if category != "":
+                category = self.category_list.index(category)
+            else:
+                category = 0
```
Comment (lhoestq) [inline datasets/urdu_fake_news/urdu_fake_news.py:96]: category `0` is "bus"; is this expected ?

**Review by lhoestq — COMMENTED** [review]
Looks really good thanks for adding this one !

I left a few comments.

Also it looks likethe dummy_data.zip file is quite heavy (500KB), could you try to reduce its size ? To do so you can take a look inside it and remove unnecessary files to only keep 1 or 2 txt files

File: datasets/urdu_fake_news/README.md (hunk 1 above)
Comment (chaitnayabasava) [reply datasets/urdu_fake_news/README.md]: made the change

File: datasets/urdu_fake_news/urdu_fake_news.py (hunk 2 above)
Comment (chaitnayabasava) [reply datasets/urdu_fake_news/urdu_fake_news.py]: No, my mistake... have updated the code for resolving this.

**Comment by chaitnayabasava** [comment]
@lhoestq looks like a lot of files were updated... shall I create a new PR?

**Comment by yjernite** [comment]
Hi @chaitnayabasava ! you can try rebasing and see if that fixes the number of files changed, otherwise please do open a new PR with only the relevant files and close this one :)

**Comment by chaitnayabasava** [comment]
Created a new PR #1230.
closing this one :)

**Pull request closed without merging**